bin/start-in-linux

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+
+declare minikube=$(which minikube);
+declare kubectl=$(which kubectl);
+declare workdir=$(cd $(dirname $0)/.. && pwd);
+
+
+start_minikube() {
+	# set the env
+	export MINIKUBE_WANTUPDATENOTIFICATION=false;
+	export MINIKUBE_WANTREPORTERRORPROMPT=false;
+	export MINIKUBE_HOME=$HOME;
+	export CHANGE_MINIKUBE_NONE_USER=true;
+	mkdir $HOME/.kube || true;
+	touch $HOME/.kube/config;
+
+	echo "[*] Starting minikube ...";
+	export KUBECONFIG=$HOME/.kube/config;
+	sudo -E ${minikube} start --vm-driver=none;
+	if [[ $? -ne 0 ]]; then
+		exit 2;
+	fi
+
+	echo -n "[*] Waiting for the cluster to become available ";
+	for i in {1..150}; do
+		${kubectl} get pods &> /dev/null;
+		if [[ $? -ne 1 ]]; then
+			break;
+		fi
+		echo -n ".";
+		sleep 2;
+	done
+	echo;
+}
+
+
+if [[ -z "${minikube}" || -z "${kubectl}" ]]; then
+	echo "minikube and/or kubectl commands not found!" > 2;
+	exit 1;
+fi
+
+if ! ${minikube} status &> /dev/null; then
+	if ! start_minikube; then
+		echo "Could not start the cluster!" > 2;
+		exit 2;
+	fi
+fi
+
+echo "[*] Applying mediawiki-containers ...";
+${kubectl} apply -f ${workdir}/mediawiki-dev.yaml;
+sleep 1;
+
+ip=$(minikube ip);
+port=$(${kubectl} get services | grep mediawiki-svc | cut -d':' -f2 | cut -d'/' -f1);
+
+echo;
+echo "Please wait until all of the containers have been started, at which";
+echo "point you can access the MediaWiki instance at http://${ip}:${port}";
```

Comment:
Might be worth specifying that the instance is actually accessible under `http://${ip}:${port}/wiki/Main_Page`. The redirect still doesn't work on OS X